util/test/demos/d3d12/d3d12_test.cpp

```diff
@@ -1023,19 +1023,18 @@ ID3DBlobPtr D3D12GraphicsTest::Compile(std::string src, std::string entry, std::
                            UTF82Wide(profile).c_str(), args.data(), (UINT)args.size(), NULL, 0,
                            NULL, &result);
 
-    if(SUCCEEDED(hr))
-    {
-      result->GetStatus(&hr);
+    HRESULT hrStatus = E_NOINTERFACE;
+    if(result)
+      result->GetStatus(&hrStatus);
```

Comment:
Can we fall back and try again without `-Qembed_debug` here? Assuming there's no better way to detect in advance whether or not this parameter is supported.

We'll want a fallback to support any dxc version rather than requiring the latest, especially as I just ran into this looking at why the nightly tests are failing and it seems to be this issue.

Reply:
Updated to try a fallback. For reference, the Windows 10 SDK 10.0.17763.0 fails with `-Qembed_debug`. The latest SDK, 10.0.19041.0, is fine. I didn't try any in between. Also, the DXC releases on Github use an entirely different versioning scheme.

Reply:
Yeh it was too much to hope for that there'd be a nice version number query or argument list or something :(.